index.js

```diff
@@ -105,23 +113,86 @@ function initApp(callback) {
     /*
      * configure routes
      */
-    app.post('/api/v1/substitution', (req, res) => {
-      if (!req.isAuthenticated() || req.user.level < config.user.level.substitute) {
-        res.status(401).send('{"error":"not authenticated or not allowed"}');
-        return;
-      }
+var Compendium = require('./lib/model/compendium');
+
+    // get all compendia for list in authorView.html
+    app.get('/api/v1/substitution/all', (req, res) => {
```

Comment:
This only exists for developing. Using the new local data path for saving the compendia i needed this to view the list of all compendia in the authorView.html
So this won't be there when the implementation of the substituter is finished.